cafebabel/templates/users/detail.html

```diff
@@ -0,0 +1,44 @@
+{% from "users/macros.html" import social_field with context %}
+{% extends 'base.html' %}
+
+{% block body_class %}profile-page{% endblock %}
+{% block body %}
+<h1>{{ user }}'s profile</h1>
+
+<section>
+  <h2>About</h2>
+  {% if user.is_me() or current_user.has_role('editor') %}
+  <a href={{ url_for('users.edit', id=user.id) }}>Edit this profile</a>
+  {% endif %}
+  {% if user.profile.has_image %}
+  <img src={{ user.profile.image_url }} alt="Picture of {{ user }}">
+  {% endif %}
+
+  <ul class=social-networks>
```

Comment:
Here we need to display icons only if the user previously added his social details. So we need a `if` statement.

Reply:
We do have `if` for each individual social field below.
A user may insert some social and not all, therefore we still need the `ul`.

Reply:
if the user doesn't fill any of these `li`, then we get an empty `ul`? > not good